### DISCUSSION TITLE
Is it possible to specify theme as a separate file?

### DISCUSSION THREAD

**Discussion by mock1328** [ClementTsang/bottom#1605] [Q&A] [answered]
I'd like to write my theme into `~/bottom/themes/theme.toml` and feed it to a `--theme` arg, but it takes enum, so I'm guessing it's not separatable from config atm?

**Comment by ClementTsang** [comment, marked as answer]
Currently not doable, sorry. Not really sure if I would implement this, but I can consider it.
